### ISSUE TITLE
Tractor Can't Line Up Under Combine Pipe

### ISSUE THREAD

**Issue by lcc1211** [Courseplay/courseplay#5316]
Map: Lone Oak
Equipment: Official KLASS DLC
CP Version: 6

I have the tractor's offset set to automatic, but the tractor thinks the pipe is shorter than it is. On fields 49 & 50 the combine went where it was supposed to, but on field 48 the tractor moves in too close to the combine. Even when I set it to manual it just ignores my setting and continues moving too close to the combine.
![20200506105013_1](https://user-images.githubusercontent.com/24764870/81192987-dc25a400-8f88-11ea-8c14-b871b1e46c6f.jpg)
[log.txt](https://github.com/Courseplay/courseplay/files/4587582/log.txt)

**Comment by pvaiko** [comment]
First of all, are you saying it worked on two fields but not on a third one? Also, what was the combine doing? Did it pull back? I see no trace of it doing that in the logs, are you sure the debug channels were on when that situation on the screenshot happened?

**Comment by lcc1211** [comment]
Yes. On fields 49 & 50 this tractor and a mod tractor were both able to line up directly under the pipe of this combine. I stopped the tractor in screenshot and drove it over to the edge of the field and selected the debug channels then started the course again.

**Comment by lcc1211** [comment]
I'll do a video and link it so you can see.

**Comment by pvaiko** [comment]
Thanks, Make sure you have the combine's movements in the log before it is full. It really shouldn't move out of the row.

**Comment by lcc1211** [comment]
Well, this time it actually works. 1st time it went under the pipe. 2nd time the tractor was manually called but it only went part of the way to the end of the row and failed to move toward the combine. I stopped the course and created a new one and it went under the pipe again.

**Comment by lcc1211** [comment]
Well, without dbug channels on things fall fall apart. When they are on everything behaves normally. Now the tractor is sitting at the return point after returning from the silo. The combine has called for the tractor but the tractor won't move. I stopped the course and started again, but it still just sits there with the engine running and the strobe on. It does turn the wheels toward the combine.

**Comment by pvaiko** [comment]
Can't do anything without the logs, and I'm sure it does not matter if the debug channels are on or not.

Looks like the tractor can't find a path to the combine, it is difficult to tell from the video why as the combine and its surroundings are not really shown.

**Comment by lcc1211** [comment]
[log.txt](https://github.com/Courseplay/courseplay/files/4588722/log.txt)
Field 48 is roughly a rectangular shape with another square at the top right. The combine was in the square.

**Comment by pvaiko** [comment]
So in this log, what exactly happened? Looks like the tractor did drive to the combine, and then something went wrong. What did you see?

**Comment by pvaiko** [comment]
Oh, now I see, that was a previous unload. I see messages it did not find a path but at that time the debug channels were deactivated...

**Comment by lcc1211** [comment]
The combine turned into a new row with about 98% of capacity filled. It stopped there and called the tractor. BTW, I moved the tractor to with a few meters of the combine and it still wouldn't move to the combine. I lined it up directly behind the combine and start the course again, but it still wouldn't move.

**Comment by pvaiko** [comment]
I got that, but we don't have the logs from that time...

**Comment by lcc1211** [comment]
That's because I just did those last maneuvers a few minutes ago. There are a number of things happening that I did not raise here. For instance, the strobe turn on when I start a CP combi offload and the engine never shuts down in waiting. It also won't shut off at the end of a fertilizing, transport or sewing course. I think I'll wait a bit before using CP again. Maybe a mod is conflicting or maybe someone else will report an issue that is similar and they will have a better explanation of the error.
